internal/hackernews/api.go

```diff
@@ -0,0 +1,68 @@
+package hackernews
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/JakeHumphries/gymshark-hacker-news/internal/models"
+	"github.com/pkg/errors"
+)
+
+// Api to get data from the hacker news api
+type Api struct{}
+
+const hackerNewsUrl string = "https://hacker-news.firebaseio.com/"
+
+// GetTopStories gets the top stories from the hacker news api
+func (a Api) GetTopStories() ([]int, error) {
+	url := fmt.Sprintf("%sv0/topstories.json?print=pretty", hackerNewsUrl)
```

Comment:
Maybe a personal preference thing but I think this would be easier to read if you dropped the last `/` from your `hackerNewsUrl` const and added it to the two places you use it:

`"%sv0/topstories.json?print=pretty"`

vs

`"%s/v0/topstories.json?print=pretty"`

Reply:
agree, much more readable.